### PR TITLE
chore: Added user command

### PR DESCRIPTION
**Pull request by ishaan-kapoor**
Now users can access the functions from their command menu without going through `:lua require('chainsaw').variableLog`.
They can do the same by `:ChainSaw variableLog` now.

This will make the plugin handy for people who prefer commands over keybindings.

I drew inspiration from [Telescope Plugin](https://github.com/nvim-telescope/telescope.nvim/blob/4626aaa2bcfdacf55fd6d44b430e2df81b2403ff/plugin/telescope.lua#L107)

## Checklist
- [x] Used only camelCase variable names.
- [x] If functionality is added or modified, also made respective changes to the
  README.md (the `.txt` file is auto-generated and does not need to be modified).
- [x] Used conventional commits keywords.

### CODE REVIEW

File: lua/chainsaw/init.lua [hunk 1]
```diff
@@ -120,5 +120,22 @@ function M.removeLogs()
...
+--------------------------------------------------------------------------------
+
+M.commandTable = {
+  assertLog = M.assertLog,
+  beepLog = M.beepLog,
+  debugLog = M.debugLog,
+  messageLog = M.messageLog,
+  objectLog = M.objectLog,
+  removeLog = M.removeLogs,
+  stackTraceLog = M.stracktraceLog,
```
Comment (chrisgrieser) [inline lua/chainsaw/init.lua:132]: here you still got the "stracktrace" typo 😛

**Review by chrisgrieser — CHANGES_REQUESTED** [review]

**Comment by chrisgrieser** [comment]
Good idea

> If functionality is added or modified, also made respective changes to the
README.md

that part of the checklist is still missing?

**Comment by ishaan-kapoor** [comment]
Should I go with the param names as I have written, or do you have better suggestions?
I will update the README accordingly.

**Comment by chrisgrieser** [comment]
Just keeping those param names for consistency is absolutely fine!

File: lua/chainsaw/init.lua (hunk 1 above)
Comment (ishaan-kapoor) [reply lua/chainsaw/init.lua:132]: pulled from main

**Comment by ishaan-kapoor** [comment]
Made the changes, please review.

File: lua/chainsaw/init.lua [hunk 2]
```diff
@@ -120,5 +120,26 @@ function M.removeLogs()
...
 
+--------------------------------------------------------------------------------
+
+M.commandTable = {
+	assertLog = M.assertLog,
+	beepLog = M.beepLog,
+	debugLog = M.debugLog,
+	messageLog = M.messageLog,
+	objectLog = M.objectLog,
+	stackTraceLog = M.stracktraceLog,
```
Comment (chrisgrieser) [inline lua/chainsaw/init.lua:131]: There is still a typo here "strack"

**Review by chrisgrieser — CHANGES_REQUESTED** [review]
Just the typo still, otherwise good to go

File: lua/chainsaw/init.lua (hunk 2 above)
Comment (ishaan-kapoor) [reply lua/chainsaw/init.lua]: fixed it

**Pull request merged**